src/wx/widgets/sdl-poller.cpp

```diff
@@ -5,7 +5,11 @@
 #include <wx/timer.h>
 #include <wx/toplevel.h>
 
+#ifndef ENABLE_SDL3
```

Comment:
Please, revert the logic here to `#ifdef ENABLE_SDL3` for clarity. Negative conditions are harder to read.